spotdl/download/downloader.py

```diff
@@ -155,7 +162,7 @@ async def download_song(self, songObj: SongObj) -> None:
         convertedFileName = convertedFileName.replace(
             '"', "'").replace(':', '-')
 
-        convertedFilePath = Path(".", f"{convertedFileName}.mp3")
+        convertedFilePath: Any = Path(".", f"{convertedFileName}.mp3")
```

Comment:
Why not something like below?
```suggestion
        convertedFilePath: Path = Path(".", f"{convertedFileName}.mp3")
```

Reply:
`convertedFilePath` is later assigned to None and mypy errors out `error: Incompatible types in assignment (expression has type "None", variable has type "Path")`

Reply:
Then make it `typing.Optional[Path]`, it's definitely better than `Any`